napari/plugins/_plugin_manager.py

```diff
@@ -42,6 +43,15 @@ class NapariPluginManager(PluginManager):
 
     def __init__(self):
         super().__init__('napari', discover_entry_point=self.ENTRY_POINT)
+
+        self.events = EmitterGroup(
+            source=self,
+            registered=None,  # emits event.value
+            disabled=None,  # emits event.added & event.removed
```

Comment:
It is confusing that a disabled event set has added and removed.

It would be more consistent to have a `enabled` event set in that case

`enabled.added == enabled`
`enabled.removed == disabled`

I feel the same dissonance as when a checkbox says `check to disable something`, the wording and the UI do not match.

Reply:
Okiedoke :) can intercept the event and rename if you want.  Those names come from EventedSet, (they're not new here).  And since this is the set of disabled plugins (which starts empty), I called it disabled.  Seems like it could be confusing both ways really

Reply:
Not sure what is the best, I just felt really confused when reading it.

I know that we are using an evented set and this is the api.

I would prefer to get a plugin name plus True (enabled) or False (disabled) as the event.value: Tuple[name, bool]



Maybe that diverges for how other places use evented set.

Reply:
@sofroniewn @ppwadhwa @jni, feel free to weigh in, I'll change it to enabled tomorrow unless anyone prefers otherwise

Reply:
I do not mean to block this PR. 

I can understand how it works and how it can be used and that is ok since it is an internal implementation detail. 🙃

Reply:
Given that `registered` emits `value=name` I think having `enabled` emit `value=Tuple[name, bool]` seems like a good choice, but my preference here is weak, so if it significantly complicates code or others have stronger preferences the alternative is fine too